bm-instance-agent/bm_instance_agent/systems/linux/driver.py

```
@@ -253,10 +254,32 @@ def detach_volume(self, instance_obj, volume_obj, volume_access_path_gateway_ips
+            self.detach_volume_from_storage_gateway(instance_obj, volume_obj, volume_access_path_gateway_ip)
+
+    def detach_volume_from_gateway(self, instance_obj, volume_obj):
+        # Get the session id
+        sid = None
+        cmd = ['iscsiadm', '-m', 'session']
+        stdout, _ = processutils.execute(*cmd)
+        iqn = None
+        if instance_obj.custom_iqn:
+            iqn = instance_obj.custom_iqn
+        else:
+            iqn = instance_obj.uuid
```

Ruff (0.8.2)

Use ternary operator `iqn = instance_obj.custom_iqn if instance_obj.custom_iqn else instance_obj.uuid` instead of `if`-`else`-block

Replace `if`-`else`-block with `iqn = instance_obj.custom_iqn if instance_obj.custom_iqn else instance_obj.uuid`

(SIM108)